_runners/vault.py

```diff
@@ -73,7 +73,10 @@ def _get_policies(minion_id, config):
 
   policies = []
   for pattern in policyPatterns:
-    policies.append(pattern.format(**mappings))
+    try:
+      policies.append(pattern.format(**mappings))
+    except KeyError:
```

Comment:
This is the error displayed for the salt policy:
`saltstack/minion/{grains[ec2_tags][Environment]}/{grains[ec2_roles]}`

With that policy configured on the salt-master, I see this error. Setting the except to ValueError rather than KeyError allows everything to operate properly.